plugin-server/src/worker/ingestion/team-manager.ts

```diff
@@ -13,6 +13,81 @@ import { getByAge, UUIDT } from '../../utils/utils'
 
 type TeamCache<T> = Map<TeamId, [T, number]>
 
+function detectPropertyDefinitionTypes(isNumerical: boolean, value: any, key: string) {
```

Comment:
It seems a bit redundant to require `isNumerical` as an argument here if we do `typeof value === 'string'` anyway.